security-council-rehearsals/templates/r1-hello-council/SignFromJson.s.sol

```diff
@@ -0,0 +1,23 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.15;
+
+// This import path should be relative to where the rehearsal script will be located after a new rehearsal is setup,
+// which is one level above the current location in
+// repo_root/security-council-rehearsals/<rehearsal-dir>/SignFromJson.s.sol
+import {SignFromJson as OriginalSignFromJson} from "../../script/SignFromJson.s.sol";
```

Comment:
I think these relative paths need an extra `../` to reach the root script. Alternatively could just use an absolute path that starts with `script/`. Currently when running `just simulate` from within `security-council-rehearsals/templates/r1-hello-council` it errors with a file not found:

```
[⠢] Compiling 9 files with Solc 0.8.15
[⠆] Solc 0.8.15 finished in 36.04ms
Error: 
Compiler run failed:
Error (6275): Source "security-council-rehearsals/script/SignFromJson.s.sol" not found: File not found. Searched the following locations: "/Users/mds/Documents/projects/superchain-ops".
ParserError: Source "security-council-rehearsals/script/SignFromJson.s.sol" not found: File not found. Searched the following locations: "/Users/mds/Documents/projects/superchain-ops".
 --> security-council-rehearsals/templates/r1-hello-council/SignFromJson.s.sol:8:1:
  |
8 | import {SignFromJson as OriginalSignFromJson} from "../../script/SignFromJson.s.sol";
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

error: Recipe `simulate` failed with exit code 1
```

Reply:
Right. The tricky thing here, which the comment above tries to clarify, is that the correct flow is what's written into the CI config file. So, it needs to compile once the this entire folder is copied into a new folder one level higher. 

It's not pretty, but this follows the convention of other rehearsals and satisfies the exisiting CI checks.

Reply:
I was confused by that comment because the path it gives is:
```
repo_root/security-council-rehearsals/<rehearsal-dir>/SignFromJson.s.sol
```
But there is no `SignFromJson.s.sol` in either the `security-council-rehearsals/<rehearsal-dir>` folders

I still am confused here though, so my original comment of:

> Currently when running `just simulate` from within `security-council-rehearsals/templates/r1-hello-council` it errors with a file not found:

Is that expected to not work?

Reply:
> But there is no SignFromJson.s.sol in either the security-council-rehearsals/<rehearsal-dir> folders

That's because those rehearsals were created prior to this PR.

Reply:
> > Currently when running `just simulate` from within `security-council-rehearsals/templates/r1-hello-council` it errors with a file not found:
> 
> Is that expected to not work?

Without a deeper refactor, I'm not sure how to make both work with the current flow used to create a new rehearsal with `just setup`. Am open to suggestions though.

Reply:
If the templates are expected to not work that is ok, just wasn't sure if it was expected behavior :)